src/game/component/SoundComponent.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+class SoundComponent
```

Comment:
This isn't a component. Read the README, the underlying system has certain requirements before a component class can actually be used as a component.